profile/README.md

```diff
@@ -1,29 +1,30 @@
-## Osmosis-Labs
+# Osmosis Labs
 
-The Osmosis-labs hosts a series of repositories relating to the Osmosis blockchain and tooling for Cosmos.
+Repositories for the Osmosis AMM and tooling for Cosmos.
```

Comment:
I think the prior sentence is more accurate.

There are other repositories for the osmosis blockchain not in this org, and its for the blockchain, not just the AMM!